modules/ROOT/pages/installation-guide.adoc

```diff
@@ -0,0 +1,5 @@
+= Installation guide
```

Comment:
please create an index page at minimum to have a convenient home

Reply:
About the index page, do I have to add it in this pr or can it be done in a subsequent one? Valentina was going to work on all of that, and all I wanted to do is just create a page for the antora start_page key.

Reply:
I guess I could create an index page and use that instead for the start_page, since I don't think it matters, we just need one to exist for the build-pr-preview.